test/feature-benchmark/mzcompose.py

```diff
@@ -320,144 +337,3 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
             f"ERROR: The following scenarios have regressions: {', '.join([scenario.__name__ for scenario in scenarios.keys()])}"
         )
         sys.exit(1)
-
-
-def workflow_mzcloud(c: Composition, parser: WorkflowArgumentParser) -> None:
```

Comment:
This workflow was added by @sploiselle , but it is a copy+paste of the default workflow with minor changes. Instead of refactoring it , I am deleting it and, if needed, we will re-introduce it in a way that avoids code duplication.

Reply:
SGTM!